content/en/tracing/setup_overview/setup/php.md

```diff
@@ -94,32 +94,43 @@ For other environments, please refer to the [Integrations][5] documentation for
 
 ### Install the extension
 
-Install the PHP extension using one of the [precompiled packages for supported distributions][5].
+Download the latest installer
 
-Once downloaded, install the package with one of the commands below.
+```shell
+curl -LO https://github.com/DataDog/dd-trace-php/releases/latest/download/datadog-setup.php
+```
+
+Run the installer
 
 ```shell
-# using RPM package (RHEL/Centos 6+, Fedora 20+)
-rpm -ivh datadog-php-tracer.rpm
+# Full installation: APM + ASM + Profiling (Beta)
+php datadog-setup.php --php-bin=all --enable-appsec --enable-profiling
 
-# using DEB package (Debian Jessie+ , Ubuntu 14.04+ on supported PHP versions)
-dpkg -i datadog-php-tracer.deb
+# APM only
+php datadog-setup.php --php-bin=all
 
-# using APK package (Alpine)
-apk add datadog-php-tracer.apk --allow-untrusted
+# APM + ASM
+php datadog-setup.php --php-bin=all --enable-appsec
+
+# APM + Profiling (Beta)
+php datadog-setup.php --php-bin=all --enable-profiling
 ```
 
-The extension will be installed for the default PHP version. To install the extension for a specific PHP version, use the `DD_TRACE_PHP_BIN` environment variable to set the location of the target PHP binary before installing.
+This command will install the extension to all the PHP binaries found in the host or container. If `--php-bin` is omitted then the installer runs in interactive mode asking the user to select the binaries for installation. The value of `--php-bin` can be a path to a specific binary in case `dd-trace-php` should be installed only to such binary.
```

Comment:
```suggestion
This command installs the extension to all the PHP binaries found in the host or container. If `--php-bin` is omitted, the installer runs in interactive mode and asks the user to select the binaries for installation. The value of `--php-bin` can be a path to a specific binary in case `dd-trace-php` should be installed only to such binary.
```